packages/markdown/src/markdown_renderers.ts

```diff
@@ -51,18 +50,19 @@ export function customHeadingRenderer(text: string, level: 1 | 2 | 3 | 4 | 5 | 6
     return `\n${' '.repeat(12)}<h${level} id="${headingId}"><a href="#${headingId}"></a>${headingText}</h${level}>`;
 }
 
-export function parseRepoName(gitRepoUrl: string): string {
-    const parsedRepoUrl = gitUrlParse(gitRepoUrl);
+export function getRepoFullName(gitRepoUrl: string): string {
```

Comment:
This is a breaking change, please always try to not do breaking changes for a public package like this one when you don't have to. I know that this is a better naming but if the returned value is still the same there is no reason to break it for everyone who potentially uses this function.

Reply:
Good catch - I totally didn't consider it!